Code/Tools/ProjectManager/Source/ScreenWidget.h

```diff
@@ -54,7 +54,9 @@ namespace O3DE::ProjectManager
         virtual void GoToScreen([[maybe_unused]] ProjectManagerScreen screen)
         {
         }
-
+        virtual void HookConnections()
```

Comment:
nit: I would prefer to call this something more generic like `Init()` as it could be used for more than just setting up connections, but am fine with calling it this for now.